esmvaltool/diag_scripts/autoassess/plot_autoassess_stratosphere.py

```diff
@@ -0,0 +1,62 @@
+"""autoassess stratosphere diagnostic."""
+import os
+import logging
+import inspect
+import sys
+
+import iris
+import yaml
+
+logger = logging.getLogger(__name__)
+
+# Diagnostic that takes two models (control_model and exp_model
+# and observational data (ERA-Interim and MERRA)
+
+
+def get_cfg():
```

Comment:
Please import this function from `esmvaltool.diag_scripts.shared` instead of copying it.

Reply:
okey dokey

Reply:
done, man